FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/HealthCheckBackgroundService.cs

```diff
@@ -53,6 +57,7 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
 
                     // Todo: Send notification to mediator and remove listeners here.
                     var listenerTasks = _healthCheckListeners.Select(l => l.ProcessHealthStatusAsync(healthStatus, cancellationToken)).ToList();
+                    _metricsLogger.LogHealthStatusMetric(healthStatus.Status == Models.HealthCheckStatus.HEALTHY ? 1 : 0);
```

Comment:
Questions: The 1 indicate health and 0 is unhealth? The metric logs also contain numbers for status?